### PR TITLE
Add shellcheck github action; fix init-env

### PR DESCRIPTION
**Pull request by sebrandon1**
I was looking through our latest CI runs and I noticed there was an error:
```
11:34:02  ++ res='Server Version: 4.7.49'
11:34:02  ++ '[' -z 'Server Version: 4.7.49' ']'
11:34:02  ++ false
11:34:02  ++ '[' -ge 0 ']'
11:34:02  ./scripts/init-env.sh: line 81: [: -ge: unary operator expected
11:34:02  + oc apply -f ./scripts/operator-ce.yaml
```

So I decided to see if shellcheck has a github action available apply all of the easy fixes to our scripts.

### CODE REVIEW

**Comment by sebrandon1** [comment]
Will only flag `severity: error` level problems as failures.  Warnings will not be listed as error.

File: scripts/create-operator-bundle.sh [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 
 # Login to the registry
 docker login quay.io/testnetworkfunction
 
 # create the test operator
 rm -rf nginx-operator
 mkdir nginx-operator
-cd nginx-operator
-operator-sdk init --domain $REGISTRY_NAME --plugins helm
+cd nginx-operator || exit
```
Comment (hamadise) [inline scripts/create-operator-bundle.sh:13]: is this good ?
I think if the folder does not exist, there's no point in the next operation -operator-sdk-

**Review by hamadise — APPROVED** [review]

File: scripts/create-operator-bundle.sh (hunk 1 above)
Comment (sebrandon1) [reply scripts/create-operator-bundle.sh:13]: Yeah correct it makes the script `exit` if the folder doesn't exist.

**Review by edcdavid — APPROVED** [review]
Wow this is great! Did not know that such a tool existed.

**Pull request merged**